src/test/java/games/strategy/engine/lobby/server/login/LobbyLoginValidatorTest.java

```diff
@@ -12,10 +12,10 @@
 import org.junit.Test;
 
 import games.strategy.engine.lobby.server.LobbyServer;
-import games.strategy.engine.lobby.server.userDB.BadWordController;
+import games.strategy.engine.lobby.server.userdb.BadWordController;
 import games.strategy.engine.lobby.server.userDB.DBUser;
```

Comment:
Checkstyle: "Wrong lexicographical order for 'games.strategy.engine.lobby.server.userDB.DBUser' import. Should be before 'games.strategy.engine.lobby.server.userdb.BadWordController'."